test-cases/gutenberg/columns.md

```diff
@@ -275,3 +275,62 @@ Expected look:
 ![DarkModeEmpty](../resources/columns-dark-mode-empty.png)
 
 --------------------------------------------------------------------------------
+
+##### TC019
+
+### Columns percentage mechanism - column's preview
+
+- Add a `Columns` block
+- Expect to see bottom sheet with layout picker
+- Choose one of the options e.g: `33/33/33`
+- Select the first column
+- Open its settings
+- Change the value e.g. to  `90`
+- Expect that fill level in the preview (to the left of a slider) increased
+
+--------------------------------------------------------------------------------
+
+##### TC020
+
+### Columns percentage mechanism - column's slider input
+
+- Add a `Columns` block
+- Expect to see bottom sheet with layout picker
+- Choose one of the options e.g: `33/33/33`
+- Select the first column
+- Open its settings
+- Press text input (to the right of a slider)
+- Type value with more than 1 decimal e.g.: `55.55555` and press `Done`
+- Expect to have value with only 1 decimal
+- Close bottom sheet and switch to HTML mode
+
+--------------------------------------------------------------------------------
+
+##### TC021
+
+### Columns percentage mechanism - column's frontend
+
+- Repeat the steps from [TC021](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc021) and [TC022](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc022) within the same `Columns` block
+- Switch to HTML mode
+- Expect that first column has width equal `90` and the second `55.5`
+- Save the post and open it on web
+- Expect to see columns displayed proportionally to the set width
+
+<img src="../resources/columns-layout-picker-flow.gif" width=300 />
```

Comment:
```suggestion
##### TC021

### Columns percentage mechanism - predefined percentage values are rendered correctly on frontend

- Add a `Columns` block
- Expect to see bottom sheet with layout picker
- Choose one of the options e.g: `33/33/33`
- Switch to html mode to see if ratios are correct
- Check the preview to see if the front end reflects the column widths
- Repeat for each layout option

##### TC022

### Columns percentage mechanism - custom percentage values are rendered correctly on frontend

- Add a `Columns` block
- Expect to see bottom sheet with layout picker
- Dismiss the layout picker
- Select the first column, go to its settings
- Use the text input to set its width to a custom value, let's say: `90`
- Set the second column width to another custom value, let's say: `55.5`
- Switch to HTML mode
- Expect that first column has width: `90` and the second one: `55.5`
- Check the preview and expect to see that the front end reflects the column widths
```

Making a new suggestions, it seems to me this way it is more clear what we are testing.